templates/web/northnorthants/tracking_code.html

```diff
@@ -0,0 +1,94 @@
+[% IF c.config.BASE_URL == "https://www.fixmystreet.com" OR c.config.BASE_URL == 'https://staging.fixmystreet.com' %]
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag() {
+    dataLayer.push(arguments);
+  }
+
+  window.onload = () => {
+    const cookieBanner = document.getElementById("wncCookieBanner");
+    if (localStorage.getItem("wncCookieConsentMode") === null) {
+      cookieBanner.classList.remove("wnc-cookie-hidden");
+      cookieBanner.classList.add("wnc-cookie-show");
+    }
+
+    if (localStorage.getItem("wncCookieConsentMode") === null) {
+      gtag("consent", "default", {
+        ad_storage: "denied",
+        analytics_storage: "denied",
+        personalization_storage: "denied",
+        functionality_storage: "denied",
+        security_storage: "denied",
+      });
+    } else {
+      console.log('gtag');
```

Comment:
This presumably doesn't need to be here any more :)

Reply:
It's in the west code as well, so will remove.